Countries/FirstScreen/ListViewController.swift

```diff
@@ -7,38 +7,62 @@
 
 import UIKit
 
-class ListViewController: UITableViewController {
+class ListViewController: UITableViewController, UISearchBarDelegate, UISearchResultsUpdating {
     
     private enum Constants {
         static let heightForRow: CGFloat = 179.0
+        static let notAplicableField = "NA"
+        static let searchBarPlaceHolder = "Find Countries"
     }
     
     var countries: [CountriesApiQuery.Data.Country] = []
-    
+    var filteredCountries: [CountriesApiQuery.Data.Country] = []
+
     let refrechControl: UIRefreshControl = {
-        let refrechControll = UIRefreshControl()
-        return refrechControll
+        let refrechControl = UIRefreshControl()
+        return refrechControl
+    }()
```

Comment:
Is it necessary to write this if there are no additional properties? 
Instead of simply writing  let refrechControl = UIRefreshControl()

Reply:
agree. will change it